FGEGraphics/GraphicsHelpers/Renderer.cs

```diff
@@ -365,6 +404,58 @@ public void RenderRectangle3D(Matrix4 mat)
         GL.BindVertexArray(0);
     }
 
+
+
+    /// <summary>
+    /// Renders a Circle based on Location (Vectors Normalized Down)
+    /// </summary>
+    /// <param name="center">The Location for the circle to be rendered</param>
+    /// <param name="radius">Radius of the circle</param>
+    /// <param name="view">View to render Circle in</param>
+    public void RenderGroundCircle(Location center, float radius, View3D view)
+    {
+        // Save current states prior to render
+        bool cullFaceEnabled = GL.IsEnabled(EnableCap.CullFace);
```

Comment:
re `Should the the GL state alterations also be up to caller of the function to keep track of? (like should the caller need to set those out of the call and reset them after)`
yes that's the caller's job to configure